package/yast2-devtools.spec

```
@@ -139,6 +139,7 @@ EOF
 %{_datadir}/YaST2/data/devtools/bin/gettextdomains
 %{_datadir}/YaST2/data/devtools/bin/ycp_puttext
 %{_datadir}/YaST2/data/devtools/data/rubocop_yast_style.yml
+%{_datadir}/YaST2/data/devtools/data/rubocop-0.71.0_yast_style.yml
```

I do like like dots inside a file name, esp. when there are already underscores used around, maybe 
```suggestion
%{_datadir}/YaST2/data/devtools/data/rubocop-0_71_0_yast_style.yml
```
would look better.